### PR TITLE
Bug/db backpop

### PR DESCRIPTION
**Pull request by rdelfin**

### CODE REVIEW

File: app/models/__init__.py [hunk 1]
```diff
@@ -36,8 +38,15 @@ class User(Base):
...
-    teams = relationship('Team', secondary=user_team_table, back_populates='users')
+    _teams = relationship('Team', secondary=user_team_table, back_populates='users')
     schedules = relationship('Schedule', back_populates='user')
+
+    @property
+    def team(self) -> Optional[Team]:
+        """
+        This is a read-only property. Do not UNDER ANY CIRCUMSTANCES attempt to set this.
+        """
+        return self._teams[0] if self._teams else None
```
Comment (franspaco) [inline app/models/__init__.py]: What about the weird case teams has 2 elements... for... weird reasons?

Comment (franspaco) [reply app/models/__init__.py]: Or just yolo it? 🤔

Comment (rdelfin) [reply app/models/__init__.py]: Unique should guarantee it won't... So EEEEH

[the remark this answers is not in the document]
Comment (rdelfin) [reply app/models/__init__.py]: Btw, you can make code suggestions

**Comment by rdelfin** [comment]
Some testing I did:
```python
In [5]: user = User(name="name", email="me@me.me")

In [6]: user.team

In [7]: user.team is None
Out[7]: True

In [8]: team = Team(name="some team")

In [9]: team.users = [user]

In [10]: user.team
Out[10]: <Team(id=None, name=some team)>
```

**Pull request merged**